### ISSUE TITLE
for model testing phase

### ISSUE THREAD

**Issue by happyallday017** [Flame-Chasers/TBPS-CLIP#9]
Thank you for your work. I have two questions that I would like you to answer: 1. How does the model load the weights it has trained for testing? 2. What is the testing principle of the model? Is it by using the test set instead of the validation set?

**Comment by byougert** [comment]
Hi, @happyallday017 

Thank you very much for your feedback and for pointing this out.

In our previous version of the code, we did not provide a standalone testing (evaluation-only) mode. As the training time of TBPS-CLIP is relatively short, we initially assumed that running training would not introduce significant overhead.

However, based on your feedback, we realize that the computational cost can still be inconvenient in practice. 

To improve usability, we have now added the [evaluation](https://github.com/Flame-Chasers/TBPS-CLIP/tree/master?tab=readme-ov-file#evaluation) mode.

Specifically, after modifying [s.config.yaml](https://github.com/Flame-Chasers/TBPS-CLIP/blob/ba090bc7dcb3b2787e80f701fc34e19f9205cee4/config/s.config.yaml#L57) (e.g., setting ``ckpt_type`` to "saved" and specifying the downloaded checkpoint path in ``saved_path``), you can directly run:

``bash shell/eval.sh``

This will load the checkpoint and perform evaluation without training.

Thanks again for your valuable suggestions!

**Comment by happyallday017** [comment]
@byougert Thank you for your reply. I have one last question. The GPU configuration in my laboratory is exactly the same as yours - 4 GPUs with 4GB of memory each, model A4048G. However, during the process of reproducing your code, the results were not as good as those in the paper. Could you please guide me on what experimental details might have been overlooked?
